### ISSUE TITLE
TextField: Narrator is reading tooltip twice on Edge

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5282]
MS Fabric React TextField is not working properly on SPO tenants default serach page .../_layouts/15/search.aspx on Edge with Narrator ON. It reads the tooltip twice when using keyboard someone tries to navigate to the textfield. This problem seems to be only in the latest version of the Edge Microsoft Edge 42.17134.1.0. This does not arise in the older version of Edge 41.16299.492.0. This is happening on microsoft tenant.

**Comment by JasonGore** [comment]
TextField example page fails some label checks which may cause Narrator and Edge to fall back to other elements. In some cases placeholder is repeated, such as for the TextField with Placeholder "I am required" example.

![textfieldaria](https://user-images.githubusercontent.com/26070760/41861429-1a9feae8-7856-11e8-9346-66117db6aca7.png)

**Comment by JasonGore** [comment]
Any resolution to this item should ensure that placeholder, label and error messages all get communicated to the user properly and do not prevent each other from vocalizing (#5283).

**Comment by JasonGore** [comment]
Further analysis shows that this also occurs in the [ARIA Placeholder Text](https://www.w3.org/WAI/tutorials/forms/instructions/) example. It seems the wrong amount of placeholder is highlighted, resulting in incorrect text being spoken before the actual placeholder text. I have created a Narrator bug to track. I will close this issue for now (and reopen #5283 as a separate issue.)
